pkg/migration/controllers/migration.go

```diff
@@ -2552,3 +2601,46 @@ func getPVToPVCMappingFromPVCObjects(pvcObjects []runtime.Unstructured) map[stri
 	}
 	return pvToPVCMapping
 }
+
+func getStashCREnabledAppRegs(crdList stork_api.ApplicationRegistrationList) map[string]bool {
+	appRegsStashMap := make(map[string]bool)
+	for _, crd := range crdList.Items {
+		for _, v := range crd.Resources {
+			appKey := fmt.Sprintf("%v-%v-%v", v.Group, v.Kind, v.Version)
+			appRegsStashMap[appKey] = v.StashStratergy.StashCR
+		}
+	}
+	return appRegsStashMap
+}
+
+func getStashedConfigMap(obj *unstructured.Unstructured) (*unstructured.Unstructured, error) {
+	configMap := &unstructured.Unstructured{}
+	jsonData, err := obj.MarshalJSON()
+	if err != nil {
+		return configMap, err
+	}
+	// obj uid is not available and appending random uid will cause the next migration's get/delete of the configmap fail
+	// hence adding kind-name for cm name.
+	cmName := fmt.Sprintf("%s-%s", strings.ToLower(obj.GetKind()), obj.GetName())
```

Comment:
Ideally this should also be a helper function under pkg/utils/utils.go

Reply:
Done